### PR TITLE
IMPLEMENT: auto-grow PromptInput up to 8 lines (#237)

### LINKED ISSUE #237: Prompt Input Grow
## Context

PromptInput does not grow in size.

## Problem statement

When typing in the PromptInput in the desktop app and you go over multiple lines which is very likely, the prompt input does not grow, it stays the same size and content scrolls. This makes it hard to read what you've written. 

## Solution

As you type over multiple lines the PromptInput should grow each time you get a new line up to a maximum of 8 lines at which point content should scroll.

## Acceptance criteria

- [ ] PromptInput grows as you type over new lines
- [ ] PromptInput does not grow higher the 8 lines
- [ ] PromptInput content continues to scroll beyond the 8 lines limit
- [ ] `bun run ci` passes

### PR DESCRIPTION
**Pull request by DaveHudson**
## Summary

- IMPLEMENT: auto-grow PromptInput up to 8 lines (#237)
- REVIEW: fix broken import alias and add height-reset edge case test

Closes #237

### CODE REVIEW

File: apps/desktop/src/renderer/src/components/chat/ChatPanel.tsx [hunk 1]
```diff
@@ -166,14 +166,18 @@ export function ChatPanel({ chat }: ChatPanelProps): React.JSX.Element {
     messagesEndRef.current?.scrollIntoView({ behavior: 'smooth' });
   }, [messages]);
 
-  // Auto-grow the textarea up to a cap so prompts with a few lines
-  // don't force the user to scroll a single-line input.
+  // Auto-grow the textarea up to 8 lines; content scrolls beyond that.
+  // biome-ignore lint/correctness/useExhaustiveDependencies: input is the intended trigger; the effect reads the DOM via textareaRef
```
Comment (DaveHudson) [inline apps/desktop/src/renderer/src/components/chat/ChatPanel.tsx]: We do NOT want to biome-ignore, bad pattern. 

Also why this useEffect? We have Tailwind & Shadcn, surely there are better options than this?

Comment (DaveHudson) [reply apps/desktop/src/renderer/src/components/chat/ChatPanel.tsx]: Fixed in bfa9866.

- Auto-grow is now CSS-only via Tailwind v4's `field-sizing-content` plus a `max-h-[calc(8*1.25rem+1.25rem)]` cap and `overflow-y-auto`. Electron 35 = Chromium 134, so `field-sizing: content` is supported. The useEffect, the textareaRef, and the inline-style writes are all gone.
- The other `biome-ignore` (scroll-on-new-message) is also gone — extracted `messages.length` to a const and depend on that, which satisfies useExhaustiveDependencies without a suppression.
- Added a rule to `.sandcastle/CODING_STANDARDS.md` banning `biome-ignore` / `@ts-ignore` / `eslint-disable`. If a rule fires we fix the underlying code.

Tests: replaced the JSDOM `scrollHeight` / `style.height` assertions with className-contract checks, since height is now a pure CSS concern.
